server.js

```diff
@@ -0,0 +1,148 @@
+// TASK 2
+import express from 'express'
+import cors from 'cors'
+import Student from './models/Student.js'
+import './db_connect.js'
+import createError from 'http-errors'
+
+const app = express()
+
+app.use(express.json())
+app.use(cors())
+
+app.get("/", (req, res)=> {
+    res.send(`<h1>Hello from Students API</h1>
+    <a href="/students">/users</a>`)
+})
+
+app.get("/students", async (req, res)=> {
+    const students = await Student.find() 
+    res.json(students)
+})
+
+app.get("/students/:id", async (req, res, next) => {
+    const {id} = req.params
+    console.log(id);
+    try {
+        const student = await Student.findById(id)
+        if(!student){
+            throw new createError(
+                400,
+                "This ID doesn't exist"
+            )
+        }
+        res.json(student)
+    } 
+    catch (err) {
+        next(err)
+    }
+})
+
+// ********************************************************
+// TASK3
+app.get("/migration", async (req, res, next) => {
+    try{
+        // The updateMany() method has three mandatory parameters – the query, the update and a callback function. The callback function, in turn, has two parameters – an error (if any occurs) and the result.
+        await Student.updateMany( 
+            {
+                role: {
+                    $exists: false // key I want to update if exist
+                  }
+            }, 
+            {
+                $set:{
+                    role: "User" // key I want to set in all documents
+                }
+            },
+            (err, result) => { // callback function required by the method to response an error or a success!
```

Comment:
Actually the callback is not needed. If you omit the callback, you get the result return and can save it in a variable:
`const result = await Student.updateMany(...)` and then send it back using `res.json( result) `

But practicing some callbacks from time to time is actually a nice training too :D